backend/src/test/java/com/javahelp/backend/data/DynamoDBUserStoreTest.java

```diff
@@ -25,7 +25,7 @@ public class DynamoDBUserStoreTest {
 
     DynamoDBUserStore db = new DynamoDBUserStore(tableName, regions);
 
-    @Test
+    @Test(timeout = 5000)
```

Comment:
Sorry for reviewing late. Just a little question, large timeout limit like this is to make sure that everything won't crash (even when the execution speed is slow?)